experimental/examples/wheel/wheel_test.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2018 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import unittest
+import zipfile
+
+
+class WheelTest(unittest.TestCase):
+    def test_customized_wheel(self):
+        filename = os.path.join(os.environ['TEST_SRCDIR'],
+                                'io_bazel_rules_python', 'experimental',
+                                'examples', 'wheel',
+                                'example_customized-0.0.1-py3-none-any.whl')
+        with zipfile.ZipFile(filename) as zf:
+            self.assertEquals(
+                zf.namelist(),
+                ['experimental/examples/wheel/lib/data.txt',
+                 'experimental/examples/wheel/lib/module_with_data.py',
+                 'experimental/examples/wheel/lib/simple_module.py',
+                 'experimental/examples/wheel/main.py',
+                 'example_customized-0.0.1.dist-info/WHEEL',
+                 'example_customized-0.0.1.dist-info/METADATA',
+                 'example_customized-0.0.1.dist-info/entry_points.txt',
+                 'example_customized-0.0.1.dist-info/RECORD'])
+            record_contents = zf.read(
+                'example_customized-0.0.1.dist-info/RECORD')
+            wheel_contents = zf.read(
+                'example_customized-0.0.1.dist-info/WHEEL')
+            metadata_contents = zf.read(
+                'example_customized-0.0.1.dist-info/METADATA')
+            # The entries are guaranteed to be sorted.
+            self.assertEquals(record_contents, b"""\
```

Comment:
Is there any particular procedure we can document for generating the file contents, e.g. in case a bugfix causes the hashes to change? I suppose if it's just build the wheel, unzip, and copy/paste, then no need.

Reply:
Indeed, it's just build and copy/paste.